test/helpers/policygen/models.go

```diff
@@ -529,7 +536,9 @@ func (t *TestSpec) CreateCiliumNetworkPolicy() (string, error) {
 	  "kind": "CiliumNetworkPolicy",
 	  "metadata": {
 		"name": "%[1]s",
-		"test": "%[3]s"
+		"labels": {
+			"test": "policygen"
+		}
```

Comment:
Do we need to also edit https://github.com/eloycoto/cilium/blob/5d28eac43263a05344b7e974214c4ac77dcc3543/test/helpers/policygen/models.go#L638 to match the number of `%*` here?

Reply:
Nope, it's not needed at all ;-)

Reply:
Done.